framework/src/modules/network/network.js

```diff
@@ -216,11 +222,15 @@ module.exports = class Network {
 					sanitizedProcedure,
 					request.data
 				);
-				this.logger.info(`Responsed to peer request ${request.procedure}`);
+				this.logger.debug(
```

Comment:
This one should happen very often so maybe it should be `logger.info(...)`?
Also request above is `info` so the response is even less important so should be `info`.

Reply:
if it happens very often shouldn't it be `trace`? otherwise we will see so many logs at info and debug levels, @fix whats your thought here?

Reply:
changed it to trace